sbin/spark-daemon.sh

```diff
@@ -146,13 +176,11 @@ run_command() {
 
   case "$mode" in
     (class)
-      nohup nice -n "$SPARK_NICENESS" "${SPARK_HOME}"/bin/spark-class $command "$@" >> "$log" 2>&1 < /dev/null &
-      newpid="$!"
+      execute_command nice -n $SPARK_NICENESS ${SPARK_HOME}/bin/spark-class $command $@
```

Comment:
If `SPARK_HOME` contains spaces, this will break. I recommend quoting both `SPARK_HOME` and `SPARK_NICENESS` as they were before.